cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -4499,6 +4499,111 @@
           ]
         }
       }
+    },
+    "CloudWatchDashboardSubstack": {
+      "Type": "AWS::CloudFormation::Stack",
+      "Properties": {
+        "Parameters": {
+          "CreateFSXSubstack": {
```

Comment:
I'd put the parameters in a logical order, if possible, according to the position in the dashboard.

Reply:
Done